Formula/harfbuzz.rb

```diff
@@ -12,46 +13,40 @@ class Harfbuzz < Formula
     sha256 "9ca4eaac80b2db11439e865f977d6b7b330b2bd5ca8f558e23be49acda67b23e" => :high_sierra
   end
 
-  head do
-    url "https://github.com/behdad/harfbuzz.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-    depends_on "ragel" => :build
-  end
-
   depends_on "gobject-introspection" => :build
+  depends_on "libtool" => :build
```

Comment:
Is `libtool` needed with meson?

Reply:
During the first testing run both the 10.15 and 10.14 tests passed, but the 10.13 tests had a lot of `brew test` failures with the following output:
```
Reason: Incompatible library version: {something}.dylib requires version 20601.0.0 or later, but libharfbuzz.0.dylib provides version 0.0.0
```
So I incorrectly thought that perhaps it was because I had removed the `libtool` dependency. 

I noticed than [`handbrake`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/handbrake.rb) has both `meson` and `libtool`, so though I would try it.

I am afraid I do not know how to fix this error on 10.13.

It also seems that the 10.15 and 10.14 tests have stalled after I made this change.